hedera-node/src/main/resources/throttles-dev.json

```diff
@@ -46,11 +46,15 @@
             "burstPeriod": 10,
             "throttleGroups": [
                 {
-                    "opsPerSec": 200,
+                    "opsPerSec": 2000,
                     "operations": [ "CryptoCreate" ]
                 },
                 {
-                    "opsPerSec": 500,
+                    "opsPerSec": 25,
+                    "operations": [ "FileCreate" ]
```

Comment:
I would bump `opsPerSec` for `FileCreate` higher so we don't have to worry about it again in the future. Let's make it 1000.

Reply:
@qnswirlds as it takes minimum of 3 bucket values, I have bumped all 3 buckets values to 1000. Thanks
```
FileCreate: min{6.25 tps (ThroughputLimits), 5.00 tps (PriorityReservations), 6.25 tps (DevCreationLimits)}
```

Reply:
Makes sense!